test/cpu_fusion.cpp

```diff
@@ -2075,3 +2076,45 @@ TEST(cpu_fusion, fuse_batch_dot_forward)
         EXPECT_TRUE(test::all_close(cpu_results.at(i), int_results.at(i), 1.0e-4f, 1.0e-4f));
     }
 }
+
+TEST(cpu_fusion, fuse_rnn_across_layer)
+{
+    pass::Manager pass_manager;
+    pass_manager.register_pass<runtime::cpu::pass::LSTMFusion>();
+    pass_manager.register_pass<runtime::cpu::pass::RNNFusion>();
+    pass_manager.register_pass<ngraph::pass::AlgebraicSimplification>();
```

Comment:
Why do we need algebraic simplification pass here? Add comment.

Reply:
when we fuse RNN and feed its output to the next RNN layer. we have the below slice->concat thing.
<img width="558" alt="screen shot 2018-06-12 at 10 39 58 am" src="https://user-images.githubusercontent.com/22088076/41307068-ff8d35f4-6e2c-11e8-93b2-f922d9399dd0.png">

One of the registered pass in Algebraic simplification pass basically looks in to {GOE->slices->concat-> RNN} and replace them with {GOE->RNN}, this helps to simply the RNN multi layer fusion pass.